src/mpyl/schema/mpyl_config.schema.yml

```diff
@@ -93,24 +86,13 @@ definitions:
                 type: string
   Kubernetes:
     type: object
-    additionalProperties: false
     required:
       - defaultCluster
       - clusters
       - deploymentStrategy
     properties:
       deploymentStrategy:
         $ref: k8s_api_core.schema.yml#/definitions/io.k8s.api.apps.v1.DeploymentStrategy
-      deployAction:
-        type: string
-        pattern: ((HelmDeploy)|(HelmDryrun)|(HelmTemplate)|(KubectlManifest))|(!ENV.*)
-        description: >-
-          Defines the action to take in the deploy step. Defaults to helmDeploy.
-          If set to HelmDeploy, the helm chart will be deployed to the cluster.
-          If set to HelmDryRun, a helm chart deployment to the cluster will be simulated.
-          If set to HelmTemplate, the helm chart will be rendered and the result will be written to the folder specified in the helmTemplateOutputPath property
-          If set to KubectlManifest, a k8s manifest be written to the file specified in the kubectlManifestOutputPath property. This manifest can be deployd with kubectl apply -f <manifest>
-        default: HelmDeploy
       outputPath:
```

Comment:
this one can also go

Reply:
Also done ced8616f3393ed8bdd62d5001bda57ae738fac87.